Failed to build on macOS with sdk 2022.1 and earlier versions.

### Error Output

```text
** BUILD FAILED **
note: Building targets in dependency order

/Users/runner/work/setup-wwise/setup-wwise/Test/SoundEnginePlugin/Test_Mac_shared.xcodeproj: error: No signing certificate "Mac Development" found: No "Mac Development" signing certificate matching team ID "BCB4VLKTK5" with a private key was found. (in target 'Test' from project 'Test_Mac_shared')
```

### Reason

This is an known issue and have been fixed in Wwise 2023.1. Audiokinetic moded premake use their apple team id by default, which cause a signing issue when build plugins on macOS with sdk 2022.1 and earlier versions. 

> see: https://www.audiokinetic.com/qa/10816/wwise-up-air-build-failed-while-trying-build-custom-plug-mac

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: [2023.1, 2023.1.2]
+        version: [2022.1.16, 2022.1, 2023.1]
     steps:
       - uses: actions/checkout@v4
 
@@ -49,7 +49,7 @@ jobs:
     runs-on: windows-latest
     strategy:
       matrix:
-        version: [2023.1, 2023.1.2]
+        version: [2022.1.16, 2022.1, 2023.1]
     steps:
       - uses: actions/checkout@v4
 
@@ -85,7 +85,7 @@ jobs:
     runs-on: macos-latest
     strategy:
       matrix:
-        version: [2023.1, 2023.1.2]
+        version: [2022.1.16, 2022.1, 2023.1]
     steps:
       - uses: actions/checkout@v4
 

--- a/README.md
+++ b/README.md
@@ -2,6 +2,8 @@
 
 Set up your GitHub Actions workflow with a specific version of Wwise SDK.
 
+> Minimum supported Wwise SDK version: `2022.1.16`
+
 ### Inputs
 
 | Name            | Description                                                                                                                                    |

